modules/nextflow/src/main/groovy/nextflow/executor/local/LocalTaskHandler.groovy

```diff
@@ -131,7 +131,7 @@ class LocalTaskHandler extends TaskHandler implements FusionAwareTask {
     }
 
     protected ProcessBuilder localProcessBuilder() {
-        final cmd = new ArrayList<String>(BashWrapperBuilder.BASH) << wrapperFile.getName()
+        final cmd = new ArrayList<String>(BashWrapperBuilder.ENV_BASH) << wrapperFile.getName()
```

Comment:
This makes sure that `.command.run` gets executed correctly on a host without `/bin/bash`.